examples/mysql/README.md

```diff
@@ -42,10 +42,14 @@ helm upgrade -i kb-addon-mysql kubeblocks-addons/mysql --version 0.9.1 -n kb-sys
 ## Examples
 
 ### [Create](cluster.yaml) 
-Create an mysql cluster with specified cluster definition 
+Create an mysql cluster in syncer mode with specified component definition 
```

Comment:
1. "a MySQL cluster"
2. it's better not expose the concept "syncer" to end user, since it's an internal project.

Create a MySQL cluster with the built-in HA manager with specified component definition